GitVersion.yml

```diff
@@ -0,0 +1,6 @@
+mode: ContinuousDelivery
+branches: {}
+tag-prefix: '[rR]'
```

Comment:
how do you tag a new major version here? I presume it would go from 15.2 to 15.3 to 15.3, etc.

manually tag the PR?

Reply:
With how things are [currently configured](https://gitversion.net/docs/reference/modes/continuous-delivery) the version is calculated based on the Git tags [here](https://github.com/gwdevhub/gwlauncher/tags). The version should be incremented each time a new tag is created. When I [ran the pipeline](https://github.com/StyxenZ/gwlauncher/actions/runs/9822898199/job/27120120430) on a fork of master, the version produced was `15.2.1`.

Reply:
I believe the addition of `increment: Minor` to the `GitVersion.yml` file would produce `15.3` for consistency with the existing releases. At the moment it looks like it's incrementing the patch version, if that's an issue.

Reply:
Hmm, I'll have to check how to keep following the current versioning scheme then. Major bumps for new features or changes, minor bumps for bug fixes or small features.

Reply:
> I believe the addition of `increment: Minor` to the `GitVersion.yml` file would produce `15.3` for consistency with the existing releases. At the moment it looks like it's incrementing the patch version, if that's an issue.

Yes, please change this to increment: Minor

New Major versions can be tagged by committing with the text `+semver: major`

Reply:
Thanks, updated.